operator-framework/src/test/java/io/javaoperatorsdk/operator/dependent/primarytosecondaydependent/PrimaryToSecondaryDependentIT.java

```diff
@@ -21,7 +21,7 @@ class PrimaryToSecondaryDependentIT {
 
   public static final String TEST_CR_NAME = "test1";
   public static final String TEST_DATA = "testData";
-  public @RegisterExtension LocallyRunOperatorExtension operator =
+  public @RegisterExtension static LocallyRunOperatorExtension operator =
```

Comment:
I would go with `@RegisterExtension public static` and fix also the previous line

Reply:
absolutely, good catch